docker-compose.yml

```diff
@@ -23,6 +23,7 @@ services:
       args:
         - BUILD_ENV=production
         - RUBY_ENV=production
+        - NODE_ENV=production
```

Comment:
You might need to add this ARG to the Dockerfile as well so that it can read
https://docs.docker.com/compose/compose-file/build/#args